app/components/upload-app/status/index.hbs

```diff
@@ -0,0 +1,126 @@
+{{!@glint-nocheck: or, did-insert}}
+<AkStack
+  {{will-destroy this.removeSubmissionCounterObserver}}
+  @spacing='1.5'
+  @alignItems='center'
+>
+  {{#if (or this.hasSortedSubmissions this.uploadedApps.files.length)}}
+    <div local-class='show-border'></div>
+
+    <AkLoader
+      data-test-uploadAppStatus-loader
+      @size={{30}}
+      @thickness='5'
+      local-class='{{this.submissionNumbers.loaderColor}} cursor-pointer'
+      @progress='100'
+      @variant={{this.submissionNumbers.variant}}
+      {{did-insert this.registerAnchorRef}}
+    >
+      <:label>
+        <AkIcon
+          data-test-uploadAppStatus-icon
+          @iconName='file-upload'
+          @variant='outlined'
+          @size='small'
+        />
+      </:label>
+    </AkLoader>
+  {{/if}}
+</AkStack>
+
+{{#if (or this.hasSortedSubmissions this.uploadedApps.files.length)}}
+  <AttachPopover
+    @interactive={{true}}
+    @arrow={{true}}
+    @placement='bottom'
+    @lazyRender={{true}}
+    @showOn='click'
+    @hideOn='clickout click'
+    @animation='shift'
+    @class='ember-attacher-popper {{local-class "popper"}}'
+    @popperTarget={{this.anchorRef}}
+  >
+    <AkStack
+      @width='full'
+      @direction='column'
+      local-class='notification-dropdown-container'
+      data-test-uploadAppStatusPopover-container
+    >
+      <AkStack @width='full' @justifyContent='space-between' class='p-2'>
+        <AkStack @spacing='1'>
+          <AkIcon @iconName='show-chart' />
+
+          <AkTypography
+            data-test-uploadAppStatusPopover-title
+            @variant='subtitle1'
+          >
+            {{t 'uploadStatus'}}
+          </AkTypography>
+        </AkStack>
+
+        <AkStack @spacing='2'>
+          <AkStack @alignItems='center'>
+            <AkIcon
+              @iconName='downloading'
+              @size='small'
+              local-class='download-icon-color'
+            />
+
+            <AkTypography
+              data-test-uploadAppStatus-submissionCount='running'
+              @variant='body2'
+              local-class='ml-half'
+            >
+              0{{this.submissionNumbers.runningSubmissions}}
```

Comment:
this won't work in case of double digits instead use
`String(this.submissionNumbers.runningSubmissions).padStart(2, 0)`